src/pyOpenMS/tests/unittests/test_ProteaseDigestion.py

```
@@ -0,0 +1,56 @@
+     ProteaseDigestion.getMissedCleavages()
+     ProteaseDigestion.setMissedCleavages()
+    """
+
+    ff = pyopenms.ProteaseDigestion()
+
+    assert pyopenms.ProteaseDigestion().setEnzyme is not None
+    ff.setEnzyme("Trypsin/P")
+    assert ff.getEnzymeName() == "Trypsin/P"
+
+    assert pyopenms.ProteaseDigestion().digest is not None
+    seq = AASequence.fromString("MHARLVP")
```

Ruff (0.12.2)

Undefined name `AASequence`

(F821)